src/api/userActionTime/services.py

```diff
@@ -13,6 +13,8 @@
 from sqlalchemy import func, literal_column, and_
 
 def create_or_update_user_action_time(date: str, duration: float, id_user: int, id_action: int):  
+    if duration >= 0 or duration <= 24:
+        abort(400, description="La durée doit être comprise entre 0 et 24 heures.")
```

Comment:
prefere error message in english
```suggestion
        abort(400, description="Duration must be between 0 and 24.")
```